app/assets/stylesheets/application/organisms/_header.scss

```diff
@@ -0,0 +1,28 @@
+.header {
+  @include container;
+  box-sizing: border-box;
+  display: table;
+  padding: 8px;
+  width: 100%;
+}
+
+#pages-home .header {
+  background: linear-gradient(to bottom, rgba(0, 0, 0, .3), rgba(0, 0, 0, .0));
```

Comment:
`.3` should be written with a leading zero as `0.3`<br>`.0` should be written with a leading zero as `0.0`